lib/creator/index.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+function replaceGeneratorName(name) {
```

Comment:
Should this be the real implementation? Export it and import here to remove code duplication, please.